main.js

```diff
@@ -49,6 +50,9 @@ function addPassword(service, password) {
   label.type = 'text';
   label.size = 30;
   label.setAttribute('autocomplete', 'off');
+  label.className = 'form-control';
+  label.setAttribute('readonly', 'true');
+  label.setAttribute('placeholder', 'Read-Only');
```

Comment:
What's your reasoning for preventing the user from altering the service name?

Reply:
Mainly I thought if someone were to open/save their password. and were to accidentally edit/delete it.

Reply:
I wanted to make it easy for users to correct typos made in the "Add Password" bit, which is why I made this an input.

Reply:
Oh, Okay yeah I good point I'll change it. I didn't think of it that way.